src/main/app/src/components/hakutulos/hakutulosSuodattimet/KoulutusTyyppiSuodatin.js

```diff
@@ -78,10 +79,37 @@ const KoulutusTyyppiSuodatin = observer(() => {
     hakuStore.searchOppilaitokset();
   };
 
+  const SelectedKoulutustyypit = () => {
+    let selectedKoulutustyypitStr = _.map(
+      valitutKoulutusTyypit,
+      `name.${i18n.language}`
+    );
+    selectedKoulutustyypitStr = _.join(selectedKoulutustyypitStr, ', ');
```

Comment:
Nämä voisi omasta mielestä vaan ketjuttaa ja käyttää vanilla JS:ää.
```suggestion
    const selectedKoulutustyypitStr = valitutKoulutusTyypit.map((koulutusTyyppi) => koulutusTyyppi[name][i18n.language])
      .join(', ')
```

Reply:
Korjaan sillä:
`    const selectedKoulutustyypitStr = valitutKoulutusTyypit
      .map((kt) => kt?.['name']?.[i18n.language])
      .join(', ');`